Physics3D/geometry/triangleMesh.cpp

```diff
@@ -136,35 +140,35 @@ EditableMesh::EditableMesh(MeshPrototype&& mesh) noexcept : MeshPrototype(std::m
 
 void EditableMesh::setVertex(int index, Vec3f newVertex) {
 	assert(index >= 0 && index < vertexCount);
-	size_t offset = getOffset(vertexCount);
-	this->vertices[index] = newVertex.x;
-	this->vertices[index + offset] = newVertex.y;
-	this->vertices[index + 2 * offset] = newVertex.z;
+	size_t correct_index = (index/8)*8*2+index;//getOffset(vertexCount);
+	this->vertices[correct_index] = newVertex.x;
+	this->vertices[correct_index + 8] = newVertex.y;
+	this->vertices[correct_index + 2 * 8] = newVertex.z;
 }
 void EditableMesh::setVertex(int index, float x, float y, float z) {
 	assert(index >= 0 && index < vertexCount);
-	size_t offset = getOffset(vertexCount);
-	this->vertices[index] = x;
-	this->vertices[index + offset] = y;
-	this->vertices[index + 2 * offset] = z;
+	size_t correct_index = (index/8)*8*2+index; //getOffset(vertexCount);
```

Comment:
Minor nitpicks on the coding style. 
our variables are formatted in camelCase, also, I'd add a comment explaining the *8*2+index, it threw me for a loop before I understood.